### ISSUE TITLE
About acc！！！

### ISSUE THREAD

**Issue by luckydog11** [xujiamu123/X2-Softmax#1]
The performance of the methods compared in the paper is much lower than that in the original text. What do you think? All performance is much lower than in the original text. Why not compare these methods（adaface or so）. Looking forward to your reply.

**Comment by xujiamu123** [comment]
Hi! For your first question, I recognize that some of the experimental results are lower than those in the original paper, but there is also some other part of the experimental results better than in the original paper such as arcface. I think it is possible due to the training set or model we use, hyperparameters, or even random seeds...Therefore I have re-experimented all loss functions to minimize the effects of the above factors. Maybe you'll run a better result than in my paper with the code :D.
For adaface, I didn't make the comparison for a simple reason that I haven't read this paper... Thanks for the recommendation, I will read it and experiment with it.
